etna/models/nn/mlp.py

```diff
@@ -93,6 +99,7 @@ def step(self, batch: MLPBatch, *args, **kwargs):  # type: ignore
         :
             loss, true_target, prediction_target
         """
+        self._validate_batch(batch)
```

Comment:
It seems we validate twice.

We can call forward pass in `step`

Reply:
As I understand, `step` isn't called during forecasting, it is called only during training.

Reply:
We use `step` in `training_step` and `validation_step`, it is for training.